config/routes.rb

```diff
@@ -8,5 +8,14 @@
       resources :likes
     end
   end
+
+  namespace 'api' do
+    namespace 'v1' do
+      resources :posts, only: [:index] do
+        resources :comments, only: [:create, :index]
+      end
+    end
+  end
+
```

Comment:
- [ ] @Lawmsangi and @lilskyex0x Great work adding code to add and display comments and also display posts, however, I found it hard creating a new comment as it requires authentication, which you have not provided a way to add in the API, kindly consider making it possible to add new comment.